Projects/vbr_core_examples/CB_022_backstress_maxwell.m

```diff
@@ -0,0 +1,37 @@
+function VBR = CB_019_backstress_maxwell
```

Comment:
Thanks for updating the CB number! I was going to do that after the fact :) but you also need to change it here: 

```suggestion
function VBR = CB_022_backstress_maxwell
```